README.md

```diff
@@ -1,2 +1,32 @@
-# SSS
-Database Code
+# Self-Sufficiency Standard Database
+
+## Background
+[The Self-Sufficiency Standard](https://selfsufficiencystandard.org/)(SSS) was created by the Center of Women's Welfare (CWW) at the University of Washington as an alterantive to the Official Poverty Measure (OPM). The Self-Sufficiency Standard data is spread across the CWW website and this repository creates a database to hold the Self-Sufficiency Standard data. 
+
+## Setting Up Computer
+It's important to set up your machine these [instructions](https://neurohackademy.org/setup/).
```

Comment:
We may change this link because we're going to recommend using WSL for windows users. Also, we might also want to point to miniconda rather than the full anaconda install.

Reply:
Let's use the links in the google doc that Priyana put together.